tests/Tests.AzureAppConfiguration/Tests.AzureAppConfiguration.csproj

```diff
@@ -12,6 +12,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="1.1.0" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.0.0" />
```

Comment:
>Microsoft.Extensions.Configuration.Json [](start = 31, length = 39)

Where is this package being used?

Reply:
It's used to load config settings from JSON file ( new ConfigurationBuilder().AddJsonFile(jsonFilePath) )

---
In reply to: [453141525](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r453141525) [](ancestors = 453141525)